.github/workflows/java-maven-openjdk-codeql.yml

```diff
@@ -0,0 +1,68 @@
+name: "CodeQL"
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: |
+          The type of machine to run the job on. Must be provided as a stringified list (e.g. `runs-on: '["ubuntu-latest","self-hosted"]'`)
+        required: true
+        type: string
+      mvn-version:
+        description: |
+          The Maven version used for the execution. You can specify minor or patch version (3.9 or 3.9.1). Default : 3.9
+        required: false
+        type: number
+        default: 3.9
+      mvn-arguments:
```

Comment:
I changed a bit the arguments to avoid repetition, this is not an extra set of arguments and not an override of the default arguments which should be fine for everyone.